src/Core/Tool/Authorizer/TosAuthorizer.php

```diff
@@ -43,6 +43,13 @@ public function isAllowed(Entity $entity, $privilege)
         // These checks are run within the user context.
         $user = $this->getUser();
 
+        //if user is not actual user user_id === 0
+        if ($privilege === 'create' &&
+            !$this->isUserOwner($entity, $user)
```

Comment:
Do you need the `isUserOwner` check?

Reply:
@rjmackay I very distinctly remember thinking through whether or not I needed it and came to the conclusion that yes, I did. Now that I look back at it, I can't see why I thought that.

Reply:
Weird. Reading the code I think that if `isUserAndOwnerAnonymous` returns `true`, then `isUserOwner` will always return `true`.